docs/sphinx/user_guide/tutorial/indexset_segments.rst

```diff
@@ -53,48 +53,62 @@ indices [0, N) with a ``RAJA::RangeSegment`` object and use it in a RAJA
 loop execution template to run a loop kernel over the range. For example:
 
 .. literalinclude:: ../../../../examples/tut_indexset-segments.cpp
-                    :lines: 114-116
+   :start-after: _rajaseq_daxpy_range_start
+   :end-before: _rajaseq_daxpy_range_end
+   :language: C++
 
 We can accomplish the same result by enumerating the indices in a 
-``RAJA::TypedListSegment`` object. Here, we assemble the indices to a standard 
-vector, create a list segment from it, and then pass the list segment to the 
+``RAJA::TypedListSegment`` object. Here, we assemble the indices in a standard 
+vector, create a list segment from that, and then pass the list segment to the 
 forall execution template:
 
 .. literalinclude:: ../../../../examples/tut_indexset-segments.cpp
-		    :lines: 132-141
+   :start-after: _rajaseq_daxpy_list_start
+   :end-before: _rajaseq_daxpy_list_end
+   :language: C++
 
-Note that we are using the following type aliases here:
+Note that we are using the following type aliases:
 
 .. literalinclude:: ../../../../examples/tut_indexset-segments.cpp
-		    :lines: 50-51
+   :start-after: _raja_list_segment_type_start
+   :end-before: _raja_list_segment_type_end
+   :language: C++
 
 Recall from discussion in :ref:`index-label` that ``RAJA::Index_type`` is
 a default index type that RAJA defines and which is used in some RAJA
 constructs as a convenience for users who want a simple mechanism to apply
 index types consistently.
 
-It is important to note what is really happening when a list segment is used. 
+It is important to understand what is happens when using list segments.
```

Comment:
"what happens" instead of "what is happens".

Reply:
Fixed. Thanks!